osu.Game/Skinning/LegacySkin.cs

```diff
@@ -292,9 +293,11 @@ private IBindable<TValue> legacySettingLookup<TValue>(SkinConfiguration.LegacySe
             }
         }
 
-        [CanBeNull]
-        private IBindable<TValue> genericLookup<TLookup, TValue>(TLookup lookup)
+        private IBindable<TValue>? genericLookup<TLookup, TValue>(TLookup lookup)
         {
+            if (lookup == null)
+                return null;
```

Comment:
Can this actually be null? More specifically `ISkin.GetConfig(null)` sounds like something that we shouldn't be supporting, and if so, it's probably better to add `where TLookup : notnull` to these cases?

Reply:
I think there was a reason this wouldn't work, but lemme try again.

Reply:
Can you try applying this one? It seems to still not be feasible due to the weird usage of `TLookup` *and* `TValue` in the same method usage... which can't both be non-nullable as far as I can tell (further up in the call stack).